packages/msgraph/package.json

```diff
@@ -27,7 +27,7 @@
     "configuration-schema.json"
   ],
   "dependencies": {
-    "@openfn/language-common": "workspace:*",
+    "@openfn/language-common": "^1.10.3",
```

Comment:
This one IS necessary, because `workspace:*` will drift to common 2.0 (and so break).

We also need a patch bump on the adaptor version.